hudi-io/src/main/java/org/apache/hudi/io/hfile/HFileReaderImpl.java

```diff
@@ -334,11 +342,11 @@ private boolean isAtFirstKeyOfBlock(BlockIndexEntry indexEntry) {
    *                        afterward
    * @param numEntries      the number of entries in the root index block
    * @param levels          the level of the indexes
-   * @return
+   * @return single/multiple-level data block index
    */
   private TreeMap<Key, BlockIndexEntry> readDataBlockIndex(HFileBlockReader rootBlockReader,
-                                                           int numEntries,
-                                                           int levels) throws IOException {
+      int numEntries,
+      int levels) throws IOException {
```

Comment:
Similar on the indentation

Reply:
Will remove the line break, not sure why my checkstyle or my auto-format is not handling this automatically.